### ISSUE TITLE
Docs site issue: Horizontal TabPanel does not fit to 320px width

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#595]
Minimally we need to fix our examples on the docs site (this does not have to be a fix to the component -- see #355 )

<dl><dt>Description</dt><dd>Content does not fit to 320px width 
Tab Panel does not fit to 320px width. 
Element: 
1. Horizontal Tabs </dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The content MUST be viewable without scrolling horizontally when the viewport is set to 320 CSS pixels wide.

HOW TO FIX: 
Fix this issue by using ANY of the following techniques:
1. Use CSS techniques such as media queries, grids, or flexbox to reflow content down to 320px width.
2. Provide options within the content to switch to a layout that does not require the user to scroll horizontally to read a line of text.

REFERENCE: 
Deque University:https://dequeuniversity.com/class/responsive-zoom2/zoom/linear-text-reflow
https://dequeuniversity.com/class/responsive-zoom2/responsive/
WCAG Understanding document:
https://www.w3.org/WAI/WCAG21/Understanding/reflow

BACKGROUND: 
People with low vision often read content on a desktop computer by scaling content using the browser zoom - up to 400%. When zooming content creates the need to scroll in the direction of reading in order to reveal lines that are cut off by the viewport, it significantly increases the effort required to read text and sometimes causes content hidden off-screen with no way to scroll or pan to it. When content is created so that it reflows at screen widths down to 320px, users can view a web page in a 1280px width browser screen and zoom the content to 400% (320px) without having to scroll side to side to read and without losing access to content.</dd><dt>Source code</dt><dd>
```html
<ul role="tablist" class="Tablist" aria-label="Horizontal Tabs"><li id="tab53" role="tab" class="Tab" tabindex="-1" aria-selected="false" aria-controls="tabpanel56">Tab 1</li><li id="tab54" role="tab" class="Tab Tab--active" tabindex="0" aria-selected="true" aria-controls="tabpanel57">Tab 2</li><li id="tab55" role="tab" class="Tab" tabindex="-1" aria-selected="false" aria-controls="tabpanel58">Tab 3</li></ul>
```
</dd><dt>Checkpoint</dt><dd>Reflow (1.4.10.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Tabs</dd></dl>

**Comment by scurker** [comment]
Docs has been rewritten since this issue was first opened, so I'm not sure if it's still relevant. Closing.
